notify/nfstatus/integration.go

```diff
@@ -112,6 +120,27 @@ func (n *statusCaptureNotifier) Notify(ctx context.Context, alerts ...*types.Ale
 	return retry, err
 }
 
+func (n *statusCaptureNotifier) recordNotificationHistory(ctx context.Context, alerts []*types.Alert, retry bool, err error, duration time.Duration) {
+	if n.notificationHistorian == nil {
+		return
+	}
+	receiverName, ok := notify.ReceiverName(ctx)
+	if !ok {
+		return
+	}
+	groupLabels, ok := notify.GroupLabels(ctx)
+	if !ok {
+		return
+	}
+	pipelineTime, ok := notify.Now(ctx)
+	if !ok {
+		return
+	}
```

Comment:
Would be good to have logs here in case of error as we don't return anything.

Reply:
done in https://github.com/grafana/alerting/pull/363/commits/bf72d2bdfd7e6a4ec8a2219e90681bc9430093e0